templates/header/topbar.html

```diff
@@ -53,15 +53,9 @@
 
                     {# The Rust dropdown menu #}
                     <li class="pure-menu-item pure-menu-has-children pure-menu-allow-hover">
-                        <a href="#" class="pure-menu-link">Rust</a>
+                        <a href="https://www.rust-lang.org/" target="_blank" class="pure-menu-link">Rust</a>
 
                         <ul class="pure-menu-children">
-                            {{ macros::menu_link(
-                                href="https://www.rust-lang.org/",
-                                text="The Rust Programming Language",
-                                target="_blank"
-                            ) }}
-
```

Comment:
I think having both links is fine, this was just about being consistent with other dropdowns

Reply:
Sounds good, I can add that back in after lunch

Reply:
One quick question, @jyn514 mentioned to remove the "Rust Programming Language" link in the original issue. Just want to have some consensus to add it back, even though it's a super small change :)

Reply:
I think we should be consistent. If we have the additional link here, we should have it for the other dropdowns. But I think it looks fine without.